src/app/components/analytics/host-analytics/host-analytics.component.html

```diff
@@ -0,0 +1,117 @@
+<section class="ev-md-container">
+  <div class="row">
+    <div class="col s12">
+      <h3 class="fs-20">Challenges Analytics Home</h3>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col s12 m5">
+      <h4 class="fs-18">Select challenge to view Analysis </h4>
+      <div>
+                <span class="main-title">
+                    Total Challenges hosted :
+                </span>
+        <span class="show-count">
+                    {{challengeListCount}}
+                </span>
+      </div>
+      <ul>
+        <li class="analytics-list analytics-team-input" *ngFor="let value  of challengeList">
+          <input type="radio" name="selectExistTeam" class="with-gap selectTeam" id="{{value.id}}" value="{{value.id}}" [(ngModel)]="challengeId">
+          <label for="{{value.id}}">
+            <div class="show-member-title pointer">{{value.title}} <span class="text-light-black">hosted by</span> {{value.creator.team_name}}
+            </div>
+          </label>
+        </li>
+      </ul>
+      <div class="mt-30 btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14 show-analytics-btn" [hidden]="challengeListCount === 0" (click)="showChallengeAnalysis()">Show Analysis &nbsp;<i class="fa fa-chevron-right"></i> </div>
```

Comment:
I saw that `[hidden]="challengeListCount === 0"` was not working when `challengeListCount` was zero. Either use `*ngIf="challengeListCount !== 0"` or you can also disable it for the above condition (like in current version).

Reply:
done